mukunzidd/project11
# #168749165 add custom route for all endpoints

#### What does this PR do?
- Add a custom route for todos endpoints
- Add auto-increment id when a new task is created 
- Fix some common irrelevant ESlint  thrown errors
#### Description of Task to be completed?
- This PR adds nothing else other than making code somehow DRY!
- It shifts todos array from ``` index.js ``` to ``` routes/todoRoute.js```
#### How should this be manually tested?
Just clone the [repo](https://github.com/mukunzidd/project11.git) locally and navigate to ``` routes/todoRoute.js ```  for running the code check [README](https://github.com/mukunzidd/project11/blob/develop/README.md)
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[PT story link](https://www.pivotaltracker.com/story/show/168749165)
#### Screenshots (if appropriate)
![contribute 2](https://user-images.githubusercontent.com/51243858/65591371-b107ba80-df8c-11e9-9ecc-5facf2141283.PNG)
![contribute 3](https://user-images.githubusercontent.com/51243858/65591373-b1a05100-df8c-11e9-964c-d268a2204cd1.PNG)
![contribute 4](https://user-images.githubusercontent.com/51243858/65591374-b1a05100-df8c-11e9-8456-15bec286839e.PNG)
![contribute1](https://user-images.githubusercontent.com/51243858/65591376-b1a05100-df8c-11e9-84e3-d016633f4b36.PNG)

#### Questions: N/A
